src/main.ts

```diff
@@ -47,13 +54,19 @@ const command: Command = {
 
 		options('c', {
 			alias: 'config',
-			describe: 'Specifies what configuration to test with: \'local\'(default), \'browserstack\', \'testingbot\', or \'saucelabs\'.',
+			describe: `Specifies what configuration to test with: 'local'(default), 'browserstack', 'testingbot', or 'saucelabs'.`,
 			type: 'string'
 		});
 
 		options('cov', {
 			alias: 'coverage',
-			describe: 'If specified coverage will be included. This is the same as adding the LcovHtml reporter'
+			describe: `If specified, additional coverage reports will be written.  The will be output to the path specified in argument '-o'/'--output'.`
+		});
+
+		options('d', {
```

Comment:
Looking through, debug feels more like `verbose` to me as it doesn't offer any ability to actually debug the tests with a debugger. Do you think it would be better named as such?